src/js/src/views/SolrWayback.vue

```diff
@@ -74,18 +74,25 @@ export default {
        to.query.urlSearch !== from.query.urlSearch ||
        to.query.grouping !== from.query.grouping ||
        to.query.facets !== from.query.facets)
+    },
+    checkIfRoutingIsPageTurn(to, from) {
+      return to.query.query === from.query.query &&
+                  to.query.imgSearch === from.query.imgSearch && 
+                  to.query.urlSearch === from.query.urlSearch && 
+                  to.query.grouping === from.query.grouping &&
+                  to.query.facets === from.query.facets &&
+                  to.query.offset !== from.query.offset
```

Comment:
I feel there is a smarter way to make this comparison but haven't figured it out yet:)